docs/HowToDebug.md

```diff
@@ -5,5 +5,5 @@ There are several debug methods.
 - [How to use RAMDUMP](../tools/ramdump/HowToUseRamdump.md)  
 - [How to use Shell Commands](../apps/system/utils/README.md)  
 - [How to use Logging System](HowToUseLoggingSystem.md)
-- [How to use Addr2line](HowToUseAddr2Line.md)
+- [How to use Addr2line tool to debug a crash](HowToDebugCrashWithAddr2lineTool.md)
```

Comment:
How about `How to debug a crash`? Because addr2line is not most important topic now. It's just a step in debug. I want to be a doc for how to get debug message on a crash and how to debug a crash using it.